Indicators/CompositeIndicator.cs

```diff
@@ -171,6 +177,12 @@ private void ConfigureEventHandlers()
                     newLeftData = null;
                     newRightData = null;
                 }
+                else
+                {
+                    // Fill forward time
+                    Current = (IndicatorDataPoint)Current.Clone();
```

Comment:
Would like to know a bit more detailed _why_ is this required:
- Why is there `repeated index values after the date 4/18`
- Why should the composite indicator FF values, he is already triggering an event when a new value is available, so it sounds more like an issue in the consumer side 🤔